jenkins/promotion/release-promotion-parallel.jenkinsfile

```diff
@@ -0,0 +1,437 @@
+                                                                                            string(name: 'DISTRIBUTION_PLATFORM', value: 'linux'),
+                                                                                            string(name: 'DISTRIBUTION_NAME', value: 'tar'),
+                                                                                            string(name: 'DISTRIBUTION_ARCHITECTURE', value: 'arm64'),
+                                                                                            string(name: 'DISTRIBUTION_BUILD_NUMBER', value: params.OPENSEARCH_RC_BUILD_NUMBER),
+                                                                                            string(name: 'INPUT_MANIFEST', value: "${params.RELEASE_VERSION}/opensearch-${params.RELEASE_VERSION}.yml"),
+                                                                                        ]
+                        echo 'Promotion successful for OpenSearch Linux tar arm64!'
+                    }
+                }
+                stage('OpenSearch Dashboards Linux tar arm64') {
+                    agent {
+                        docker {
+                            label AGENT_LINUX_X64
+                            image 'docker/library/alpine:3'
+                            registryUrl 'https://public.ecr.aws/'
+                            alwaysPull true
+                        }
+                    }
+                    steps {
+                        echo 'Triggering distribution-promote-artifacts for OpenSearch Dashboards Linux tar arm64'
+                        build job: 'distribution-promote-artifacts', wait: true, parameters: [string(name: 'DISTRIBUTION_JOB_NAME', value: 'distribution-build-opensearch-dashboards'), 
+                                                                                            string(name: 'DISTRIBUTION_PLATFORM', value: 'linux'),
+                                                                                            string(name: 'DISTRIBUTION_NAME', value: 'tar'),
+                                                                                            string(name: 'DISTRIBUTION_ARCHITECTURE', value: 'arm64'),
+                                                                                            string(name: 'DISTRIBUTION_BUILD_NUMBER', value: params.OPENSEARCH_DASHBOARDS_RC_BUILD_NUMBER),
+                                                                                            string(name: 'INPUT_MANIFEST', value: "${params.RELEASE_VERSION}/opensearch-dashboards-${params.RELEASE_VERSION}.yml"),
+                                                                                        ]
+                        echo 'Promotion successful for OpenSearch Dashboards Linux tar arm64!'
+                    }
+                }
+                stage('OpenSearch Dashboards Linux tar x64') {
```

Comment:
Hey @gaiksaya , I think the idea is to run both OS and OSD tar x64 at the end.

Reply:
I see! Is there any reason to wait for Dashboards till the end too? OpenSearch I know is for maven publishing reasons here.